### PR TITLE
PAINT-281 Colors during Eraser

### PR DESCRIPTION
**Pull request by ndrnour**
* It should be possible to change colors while someone uses the eraser. Just without any effect on the eraser itself, but on the next tool the user chooses.

### CODE REVIEW

**Comment by ndrnour** [comment]
please run espresso tests

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/EraserToolIntegrationTest.java [hunk 1]
```diff
@@ -280,6 +279,6 @@ public void testColorButtonBoolean() throws NoSuchFieldException, IllegalAccessE
 				.performSelectTool(ToolType.ERASER);
 		EraserTool eraserTool = (EraserTool) PaintroidApplication.currentTool;
 		boolean eraserSelected = getToolMemberColorButton(eraserTool).getDrawSelectedColor();
-		assertFalse("Value should be false", eraserSelected);
+		assertTrue("Value should be True", eraserSelected);
```
Comment (ThomasSchwengler) [inline Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/EraserToolIntegrationTest.java]: please rewrite to `assertTrue(eraserSelected);`, the comment is unecessary

File: Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/EraserTool.java [hunk 2]
```diff
@@ -1,79 +1,72 @@
...
- *  License, or (at your option) any later version.
- *
- *  This program is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- *  GNU Affero General Public License for more details.
- *
- *  You should have received a copy of the GNU Affero General Public License
- *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ * Paintroid: An image manipulation application for Android.
```
Comment (ThomasSchwengler) [inline Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/EraserTool.java]: please revert changes to the copyright header

File: Paintroid/src/main/java/org/catrobat/paintroid/ui/button/ColorButton.java [hunk 3]
```diff
@@ -40,7 +40,11 @@
 	private static final int RECT_BORDER_COLOR = Color.LTGRAY;
 	private static final boolean DEFAULT_DRAW_SELECTED_COLOR = true;
 
-	private Paint colorPaint;
+	public static Paint getColorPaint() {
+		return colorPaint;
+	}
+
+	private static Paint colorPaint;
```
Comment (ThomasSchwengler) [inline Paintroid/src/main/java/org/catrobat/paintroid/ui/button/ColorButton.java]: Please revert the changes to `colorPaint`, this doesn't have to be static.
`getColorPaint()` is also not needed as far as I can see.
It seems that there are some functions and variables in here that are not needed anymore:
* `resetDrawSelectedColor()`
* `getDrawSelectedColor()`
* `setDrawSelectedColor()`
* `DEFAULT_DRAW_SELECTED_COLOR`

can you remove those too?

**Pull request merged**